web-frontend/src/mirage/server.ts

```diff
@@ -16,6 +16,9 @@ function mockServer(environment = ENV.DEV) {
         nickname() {
           return faker.internet.userName();
         },
+        candidateBalance() {
```

Comment:
🤔 
```suggestion
        userBalance() {
```

Reply:
on the blockchain side, we have final balance and candidate balance, this is why we use the same word in the wallet backend API and also here

Reply:
Got it! Thanks!